src/sources/coingecko.ts

```diff
@@ -30,7 +30,7 @@ export class CoinGeckoMarketSource implements MarketSource {
         let tokenInfo: TokenInfoWithCoingeckoId = baseTokenInfo;
         let coingeckoId: string = tokenInfo.extensions.coingeckoId;
         // Hack to work around TerraUSD CoinGecko ID mismapping
-        if (coingeckoId === "terra-usd") {
+        /*if (coingeckoId === "terra-usd") {
```

Comment:
Will this be removed in the next release once we verify it works in API?

Reply:
Forgot to remove after running unit tests :+1: